data/json/itemgroups/guns.json

```diff
@@ -227,6 +227,8 @@
       { "item": "h&k416a5", "prob": 50, "charges-min": 0, "charges-max": 30 },
       { "item": "m107a1", "prob": 30, "charges-min": 0, "charges-max": 10 },
       { "item": "as50", "prob": 20, "charges-min": 0, "charges-max": 10 },
+      { "item": "dsr50", "prob": 20, "charges-min": 0, "charges-max": 3 },
```

Comment:
You'll want to change these to match your new item_id's, as well as their capacities.

Reply:
oh you right.. i forgot that